kolibri/core/content/api.py

```diff
@@ -1310,6 +1315,79 @@ def consolidate(self, items, queryset):
         return sorted_items
 
 
+class ContentDownloadRequestViewset(ValuesViewset):
```

Comment:
Probably want to use a `ReadOnlyValuesViewset` here, as that will only provide the detail and list methods - which means that only create and delete will be added, with no support for updates (which otherwise would bypass your logic and might produce unpredictable behaviour).